### ISSUE TITLE
UI Textbox background doesn't resize according to text in it.

### ISSUE THREAD

**Issue by ganimtron-10** [fury-gl/fury#407]
## Description
When using `TextBox2D` and typing into it, the background `(Rectangle2D inherited from TextBlock2D)` of the `TextBox2D` doesn't seem to resize according to the text entered into it. The background remains at the same size with which we initialized it.

## Way to reproduce
Create a TextBox and add it to the scene and then keep typing into it.

### My Code
```python
from fury import ui,window
t = ui.TextBox2D(position=(50,100),text="This is a Trial.\nFor resizeable textbox background.",color=(0.75,0.5,0),width=10,height=10)
sm = window.ShowManager()
sm.scene.add(t)
sm.start()
```

## System Info
```
{'fury_version': '0.7.0', 'pkg_path': 'C:\\Users\\Lenovopc\\AppData\\Local\\Prog
rams\\Python\\Python38\\lib\\site-packages\\fury', 'commit_hash': '58dff3a56e4b1
d80581616b0bb90a03548d3ca15', 'sys_version': '3.8.7 (tags/v3.8.7:6503f05, Dec 21
 2020, 17:59:51) [MSC v.1928 64 bit (AMD64)]', 'sys_executable': 'C:\\Users\\Len
ovopc\\AppData\\Local\\Programs\\Python\\Python38\\python.exe', 'sys_platform':
'win32', 'numpy_version': '1.18.4', 'scipy_version': '1.4.1', 'vtk_version': '9.
0.1', 'matplotlib_version': '3.2.1', 'dipy_version': '1.3.0'}
```

## Demo
![TextBox2D_Background](https://user-images.githubusercontent.com/64432063/113734205-c54b7c80-9718-11eb-8f86-ac4e0bb52f33.gif)

**Comment by skoudoro** [comment]
good catch! feel free to propose a fix!  Otherwise, we look into it.

**Comment by ganimtron-10** [comment]
Thank You @skoudoro . 
I will create a Pr soon.

**Comment by ganimtron-10** [comment]
Fixed In #442 .
